### PR TITLE
Ui updates

### PR DESCRIPTION
**Pull request by miranmirza**
The HTML and CSS were separated to clean up the code base. The cancel button on the candidate page behaviour was also modified to take you back to the candidate information page as opposed to the home page. Finally the default timezone was changed from UTC to EST.

### CODE REVIEW

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -9,3 +9,9 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( edit_new.css )
```
Comment (EmilyMB) [inline config/initializers/assets.rb:12]: @miranmirza usually all the assets in app/assets/stylesheets should be automatically loaded. Was it being fussy and not showing the styles?

**Comment by miranmirza** [comment]
The problem was that it was loading all the CSS so it would interfere at times with other pages

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -5,7 +5,6 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
   <link rel='stylesheet' href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic'>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (EmilyMB) [inline app/views/layouts/application.html.erb:8]: I've usually seen this left in to make sure that the stylesheets are all loaded. Any particular reason you wanted to take it out?

Comment (miranmirza) [inline app/views/layouts/application.html.erb:8]: It was loading a CSS file in the login page which was overriding the button style

File: app/views/candidates/edit.html.erb [hunk 3]
```diff
@@ -1,84 +1,6 @@
-<style>
```
Comment (EmilyMB) [inline app/views/candidates/edit.html.erb:1]: :+1: on separating this all out!

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (EmilyMB) [inline app/views/layouts/application.html.erb:8]: Was it just the one login button that was having issues?

Comment (miranmirza) [inline app/views/layouts/application.html.erb:8]: Yes

Comment (EmilyMB) [inline app/views/layouts/application.html.erb:8]: So perhaps just add another class on the button? login-btn or something?

**Pull request closed without merging**